client/swagger.yaml

```diff
@@ -5,7 +5,7 @@ info:
     The API is accessible through HTTP calls on specific URLs
     carrying JSON modeled data.
     The transport medium is a Unix Domain Socket.
-  version: 0.23.0
+  version: 0.24.0
```

Comment:
Can you rebase this branch? master should have the change already.

Reply:
Fixed it.